db/migrate/20190219180039_add_comments_to_authentications_table.rb

```diff
@@ -0,0 +1,58 @@
+class AddCommentsToAuthenticationsTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :authentications, "Authentication information for various resources."
+
+    change_column_comment :authentications, :id, "[builtin] The internal record ID. This is the primary key."
+    change_column_comment :authentications, :name, "Symbolic name for the authentication"
+    change_column_comment :authentications, :authtype, "The authentication type or role, e.g. ssh_keypair, default, amqp, etc."
+    change_column_comment :authentications, :userid, "The userid used in a userid + password authentication approach."
```

Comment:
The userid is the unique string identifier used in the MiQ database to identify a user record. The format of the userid varies depending on authentication mode but for all external authentication modes it is UPN format.